DEVELOPMENT.md

```diff
@@ -0,0 +1,75 @@
+
+# Developer Project setup
+
+This project uses the following:
+ - yarn classic (not yarn modern).
+ - Vite - a replacement for react-create-app.
+ - Jest testing for basic features.
+ - Cypress testing for key cross-browser features. 
+
+The NSHM project https://github.com/GNS-Science/toshi-nest is a key dependency of Kororaa, providing resuable 
+UI React conponenets. It is published in the github NPM registry, and is provate, so the following steps are 
+required to get this dependency satisfied.
+
+You will need to authenticate in order to install the `@gns-science/toshi-nest` dependency. For this:
+
+## Create a Github Personal Access Token (PAT).
+ - follow guidance in the https://github.com/GNS-Science/toshi-nest README.md.
+ - make sure your Github Personal Access Token (PAT) lifetime is <= 90days.
+ - give the PAT, SSO access got GNS_Science when you create it
+
+For more background, see the Github doc [#about-personal-access-tokens](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#about-personal-access-tokens).
+
+## Using yarn-classic yarn@1.22
+ - create the PAT as above, then
+ - either: 
+     a) login to the github NPM registry manually using:
+        `npm login --scope=@GNS-Science --registry=https://npm.pkg.github.com`
+         you must supply your github usename AND the PAT. (This will update your `~/.npmrc` file.)
+    b) edit your `~/.npmrc` file like so, using the PAT.
+        ```
+        //npm.pkg.github.com/:_authToken={YOUR-PAT-HERE}
+        @GNS-Science:registry=https://npm.pkg.github.com/
+        ```
+ - now you can run `yarn` and it will use the npm credentials for the scope `@gns-science`
+
+## Using yarn-modern
```

Comment:
I'm a bit confused as the top of this document says "yarn classic (not yarn modern)."

Reply:
@chrisdicaprio I did some work with both so I wanted to capture that here. I think we'll eventually move to modern in place of classic. But for now we're using classic.